### PR TITLE
Melissaahn/release/5.3.0 to release/5.3.0

### PR DESCRIPTION
**Pull request by melissaahn**
### Summary
Cherrypicks:
- https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/2069
- https://github.com/AzureAD/microsoft-authentication-library-for-android/commit/03b851df33c9793f8c04f28e313a8c930c54c3ad

### CODE REVIEW

File: testapps/testapp/build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-android-extensions'
 
-def msalVersion = "5.2.0"
+def msalVersion = "5.+"
```
Comment (iamgusain) [inline testapps/testapp/build.gradle:32]: lets keep it to specific version.

**Review by iamgusain — APPROVED** [review]

File: msalautomationapp/build.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
-def msalVersion = "5.0.0"
+def msalVersion = "5.+"
```
Comment (iamgusain) [inline msalautomationapp/build.gradle:4]: lets make it specific version to this release.

**Review by fadidurah — APPROVED** [review]

File: msalautomationapp/build.gradle (hunk 2 above)
Comment (melissaahn) [reply msalautomationapp/build.gradle:4]: Talked with Amit offline; keeping these to +.

File: testapps/testapp/build.gradle (hunk 1 above)
Comment (melissaahn) [reply testapps/testapp/build.gradle:32]: Talked with Amit offline; keeping these to +.

**Pull request merged**